README.md

```diff
@@ -11,9 +11,9 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 
 Current tools include:
 
-1. [Assembly diffs](doc/diffs.md): jit-diff, jit-dasm, jit-analyze.
+1. [Assembly diffs](doc/diffs.md): jit-diff, jit-dasm, jit-dasm-pmi, jit-analyze, pmi.
```

Comment:
Well, "pmi" on its own doesn't fit in the "Assembly diffs" category. Maybe it needs a:

"JIT testing tools"

category.